### ISSUE TITLE
Question for Reshape Weights?

### ISSUE THREAD

**Issue by iandecks45** [tflearn/tflearn#686]
I'm trying to use already trained network for different size inputs. For example, training shape is (-1, 20, 20, 3) and prediction shape is (-1, 500, 500, 3) and this is the network:

        network = conv_2d(x, 16, 3, activation='relu')
        network = max_pool_2d(network, 2)

        network = conv_2d(network, 32, 3, activation='relu')
        network = max_pool_2d(network, 2)

        if isTrain:
            network = fully_connected(network, 64, activation='relu', name='fc1')
            network = dropout(network, 0.5)
            network = fully_connected(network, 64, activation='relu', name='fc2')
            network = dropout(network, 0.5)
            network = fully_connected(network, 2, activation='softmax', name='fc3')
        else:
            network = conv_2d(norm2, 64, 4, activation='relu', name='fc1')
            network = conv_2d(network, 64, 1, activation='relu', name='fc2')
            network = conv_2d(network, 2, 1, activation='softmax', name='fc3')

        x = regression(network, optimizer='adam', loss='categorical_crossentropy')

while it is not train, it's giving a shape match error which is normal. My question is how I can reshape the weight of "fc1", "fc2", "fc3" according to the given size of an input?

**Comment by aymericdamien** [comment]
You must keep the same shape at training and inference time, there is not way to reshape the weights. You need to reshape your images to a same shape before feeding them to the neural net.
